README.md

```diff
@@ -13,6 +13,10 @@ It makes your competitive coding life easier :grin: by automating many things fo
 <!-- ALL-CONTRIBUTORS-BADGE:START - Do not remove or modify this section -->
 [![All Contributors](https://img.shields.io/badge/all_contributors-48-orange.svg)](#all-contributors)
 <!-- ALL-CONTRIBUTORS-BADGE:END -->
+
+[![Open Collective](https://opencollective.com/cpeditor/tiers/backer.svg?avatarHeight=36)](https://opencollective.com/cpeditor#section-contribute)
```

Comment:
This badge looks different from the others, so I think it should be above the all-contributors badge.

Reply:
Sure!